app/renderer/components/main/main.js

```diff
@@ -530,7 +533,7 @@ class Main extends React.Component {
     // used in renderer
     props.isFullScreen = activeFrame.get('isFullScreen', false)
     props.isMaximized = isMaximized() || isFullScreen()
-    props.captionButtonsVisible = isWindows()
+    props.captionButtonsVisible = isWindows
     props.showContextMenu = !!currentWindow.get('contextMenuDetail')
```

Comment:
This being cached at the top could potentially save some time (avoiding an extra function call and check for navigator)